CharlieBackend.Api/Controllers/HomeworkController.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System;
+using CharlieBackend.Core;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using System.Collections.Generic;
+using CharlieBackend.Core.DTO.Homework;
+using Microsoft.AspNetCore.Authorization;
+using Swashbuckle.AspNetCore.Annotations;
+using CharlieBackend.Business.Services.Interfaces;
+
+namespace CharlieBackend.Api.Controllers
+{
+    /// <summary>
+    /// Controller to make operations with homework
+    /// </summary>
+    [Route("api/homework")]
+    [ApiController]
+    public class HomeworkController : ControllerBase
+    {
+        private readonly IHomeworkService _homeworkService;
+
+        /// <summary>
+        /// Homework controllers constructor
+        /// </summary>
+        public HomeworkController(IHomeworkService homeworkService)
+        {
+            _homeworkService = homeworkService;
+        }
+
+        /// <summary>
+        /// Adds hometask
+        /// </summary>
+        [SwaggerResponse(200, type: typeof(HometaskDto))]
+        [Authorize(Roles = "Admin, Mentor")]
```

Comment:
Secretary cant use it?

Reply:
I think no, as secretary can not be involved in study process